The indent man page (which was pre-generated for the indent source tarball
by the texinfo2ma program), doesn't have the indent Edition, Version and
"last updated" information, so we add them in via this patch.

--- indent-2.2.9/man/indent.1.orig	2013-01-07 07:18:04.585809416 -0800
+++ indent-2.2.9/man/indent.1	2013-01-07 07:18:55.704178616 -0800
@@ -12,8 +12,8 @@
 \-\-version
 .SH "DESCRIPTION"
 This man page is generated from the file \fIindent.texinfo\fR.
-This is Edition  of "The \fBindent\fR Manual",
-for Indent Version , last updated .
+This is Edition 2.2.9 of "The \fBindent\fR Manual",
+for Indent Version 2.2.9, last updated 10 November 2002.
 
 The \fBindent\fR program
 can be used to make code easier to read.  It can also convert from one
